### PR TITLE
More spider tweaks

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Slightly better numbers

Makes spider toxins more in line with a building up toxin issue rather then crippling mid-combat
Slows down many spiders form being a bit to fast for anyone not running on speed drugs or speed buffs in a line

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
I'm likely gonna hold off on making any tweaks this soon after implementing the change. I wanna see how it actually figures out in play before making another adjustment

**Comment by Trilbyspaceclone** [comment]
Sure thing
no rush

**Comment by Yawet330** [comment]
> I'm likely gonna hold off on making any tweaks this soon after implementing the change. I wanna see how it actually figures out in play before making another adjustment

it sucks hard

like
straight up through my jorts


Real stuff though, spiders are currently so unfun we have synths who specifically meta-build to kill every spider in 1-2 shots and set webs on fire at the same time to specifically genocide every nest roundstart, specifically to stop them from becoming an issue.

**Comment by cdb-is-not-good** [comment]
> > I'm likely gonna hold off on making any tweaks this soon after implementing the change. I wanna see how it actually figures out in play before making another adjustment
> 
> it sucks hard
> 
> like straight up through my jorts
> 
> Real stuff though, spiders are currently so unfun we have synths who specifically meta-build to kill every spider in 1-2 shots and set webs on fire at the same time to specifically genocide every nest roundstart, specifically to stop them from becoming an issue.

:p So you're telling me that people are becoming pro-active and taking advantage of the many tools already provided to counter these threats?

File: code/modules/mob/living/carbon/superior_animal/giant_spider/giant_spider.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 	armor = list(melee = 5, bullet = 5, energy = 0, bomb = 5, bio = 10, rad = 25)
 
-	move_to_delay = 4
+	move_to_delay = 5
```
Comment (cdb-is-not-good) [inline code/modules/mob/living/carbon/superior_animal/giant_spider/giant_spider.dm:24]: This one can likely stay at 4  - five has been a BIT too slow.

File: code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm [hunk 2]
```diff
@@ -102,9 +103,9 @@
 	melee_damage_lower = 3
 	melee_damage_upper = 5
 	meat_type = /obj/item/reagent_containers/food/snacks/meat/spider/recluse
-	move_to_delay = 5 //slower than other nurse subtypes but faster than guardians. these are OP and annoying, so don't make them too fast.
 	meat_amount = 2
 	egg_inject_chance = 15 //Defiently
+	move_to_delay = 6 // Very slow do to being 1 hit unfun
```
Comment (cdb-is-not-good) [inline code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm:108]: these are already pretty slow- why are we nerfing them?

**Comment by cdb-is-not-good** [comment]
Aside from the two above comments, this all looks good.

**Pull request merged**